### PR TITLE
Move 'Event' to 'Announcement', minor styling changes

### PR DESCRIPTION
**Pull request by user411**
Change nomenclature across repo to move 'Event' to 'Announcement'. Minor styling adds, minor file division adds in oracle-server-ts, removed deprecated MessageTypes

### CODE REVIEW

**Comment by user411** [comment]
We'll eventually want to migrate to `npm run tests` for docker image testing. Many more tests to write, added some basic ones.

File: oracle-server-ts/lib/index.ts [hunk 1]
```diff
@@ -69,57 +71,57 @@ export function SignMessage(message: string) {
...
-  const m = getMessageBody(MessageType.createnumericevent, [eventName, maturationTimeISOString, minValue, maxValue, unit, precision])
+  const m = getMessageBody(MessageType.createnumericannouncement, [eventName, maturationTimeISOString, minValue, maxValue, unit, precision])
   return SendOracleMessage(m).then(response => {
-    // console.debug('CreateNumericEvent response:', response)
+    // console.debug('CreateNumericAnnouncement response:', response)
     return <OracleResponse<string>>response
   })
 }
 
 // Rename SignEnum()?
```
Comment (Christewart) [inline oracle-server-ts/lib/index.ts:116]: nit: this comment can go

File: oracle-server-ts/lib/index.ts [hunk 2]
```diff
@@ -146,3 +148,25 @@ export function GetSignatures(eventName: string) {
     return <OracleResponse<OracleEvent>>response
   })
 }
+
+export function DeleteAnnouncement(eventName: string) {
+  console.debug('DeleteAnnouncement()')
```
Comment (Christewart) [inline oracle-server-ts/lib/index.ts:153]: might be worth adding the name of the event we are deleting to the `console.debug()`

File: oracle-server-ts/lib/index.ts [hunk 3]
```diff
@@ -146,3 +148,25 @@ export function GetSignatures(eventName: string) {
...
+
+  const m = getMessageBody(MessageType.deleteannouncement, [eventName])
+  return SendOracleMessage(m).then(response => {
+    // console.debug('DeleteAnnouncement response:', response)
+    return <OracleResponse<OracleEvent>>response
+  })
+}
+
+export function DeleteAttestation(eventName: string) {
+  console.debug('DeleteAttestation()')
```
Comment (Christewart) [inline oracle-server-ts/lib/index.ts:164]: might be worth adding the event name of the attestation we are deleting to this log

**Review by Christewart — COMMENTED** [review]
Tested 448258db838944ede543a402bfcede9fb0370ed5

I attested to this oracle and things seemed to work fine :tada: 

https://test.oracle.suredbits.com/announcement/a95fa796c4295c99fbcdff9248c6ddd9f61271d835a36ffd195d24921b269e21

File: oracle-server-ts/lib/index.ts (hunk 1 above)
Comment (user411) [reply oracle-server-ts/lib/index.ts:111]: Done

File: oracle-server-ts/lib/index.ts (hunk 2 above)
Comment (user411) [reply oracle-server-ts/lib/index.ts]: Done

File: oracle-server-ts/lib/index.ts (hunk 3 above)
Comment (user411) [reply oracle-server-ts/lib/index.ts]: Done

**Pull request merged**